rpc/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![allow(clippy::arithmetic_side_effects)]
 mod cluster_tpu_info;
+mod common;
```

Comment:
Instead of this generically named new module, how about we create an `rpc` directory and put the above module in there, named something specific (possibly just the name of the method)? Then the import in rpc.rs looks like `use crate::<NEW_MODULE>::get_account_from_overwrites_or_bank` (see runtime/src/bank.rs for an example of this organizational approach)
If I get motivated, I might move some of the other helpers cluttering up rpc.rs into that mod directory as well.

Reply:
it looks like [18f7969](https://github.com/solana-labs/solana/pull/34619/commits/18f7969b8d3e202129feddfa1c6e663d6ec72ba4) now.